core/embed/rust/src/ui/layout/simplified.rs

```diff
@@ -0,0 +1,129 @@
+#[cfg(feature = "button")]
+use crate::trezorhal::io::io_button_read;
+#[cfg(feature = "touch")]
+use crate::trezorhal::io::io_touch_read;
+#[cfg(feature = "button")]
+use crate::ui::event::ButtonEvent;
+#[cfg(feature = "touch")]
+use crate::ui::event::TouchEvent;
+use crate::ui::{
+    component::{Component, Event, EventCtx, Never},
+    constant::SCREEN,
+    display,
+};
+use num_traits::ToPrimitive;
+
+#[cfg(feature = "backlight")]
+use crate::ui::model::theme::{BACKLIGHT_DIM, BACKLIGHT_NORMAL};
```

Comment:
This breaks `make test_rust` on model R because `backlight` feature is enabled together with `model_tr`. We could reconfigure the features but relying on these constants is wrong in general -- what if on TS5 we want to fade to user-defined value instead of a built-in constant?

Short-term, we should remove this `use`, and refer to `crate::ui::model_tt::theme::XYZ` in the `fadein()` and `fadeout()` functions, probably conditional on both `backlight` _and_ `model_tt`.

Longer-term, I'm not sure yet. Something something `Model` trait that carries the constants and all these simplified functions, with implementations for each model?